src/book/chapter.rs

```diff
@@ -1,24 +1,44 @@
-use std::path;
+use std::path::{Path, PathBuf};
+use book::metadata::Author;
 
 /// The Chapter struct holds the title of the chapter as written in the SUMMARY.md file,
 /// the location of the markdown file containing the content and eventually sub-chapters
+
+/// TODO use in template: author, description, index, class
+
 #[derive(Debug, Clone)]
 pub struct Chapter {
+    /// The title of the chapter.
     title: String,
-    file: path::PathBuf,
+    /// Path to chapter's markdown file.
+    file: PathBuf,
+
+    /// TODO The author of the chapter, or the book.
+    author: Author,
+    /// TODO The description of the chapter.
+    description: String,
+    /// TODO Index number of the chapter in its level. This is the Vec index + 1.
+    index: i32,
+    /// TODO CSS class that will be added to the page-level wrap div to allow customized chapter styles.
+    class: String,
```

Comment:
I am uncomfortable with the idea of storing a class attribute in the chapter struct. Because the notion of class is **very** renderer specific. Currently it would be OK, but what happens when we have more renderers? I think chapter should only contain information that is generic over the renderers.

We have already discussed the notion of chapter specific configurations, using YAML headers or another method. I think per-chapter styling would fit better in those configurations.